apps/qubit/modules/term/actions/indexAction.class.php

```diff
@@ -93,10 +93,11 @@ public function execute($request)
       $this->forward404();
     }
 
-    // Check that this isn't the root
-    if (!isset($this->resource->parent))
```

Comment:
We should not remove this check, I'd keep it in a different `if` like it was.

Reply:
Fixed! Sorry, forgot to not rebase my fix.